README.md

```diff
@@ -66,3 +70,23 @@ to deploy it on your workstation. (You'll still run
   ```bash
   INLUNCH_ANSWERS=my_answers.yml INLUNCH_FQDN=my_machine.example.org ./instack-virt.sh
   ```
+  
+* After installing the undercloud an overcloud might be deployed.
+
+  Clone the tripleo-heat-templates repository if used THT from sources:
+  
+    ```bash
+  git clone https://github.com/openstack/tripleo-heat-templates
+  ```
+  Then deploy the overcloud:
+  
+  ```bash
+  openstack overcloud deploy \
+  --libvirt-type qemu \
+  --ntp-server clock.redhat.com \
```

Comment:
Also i wonder whether the overcloud deployment instructions should be part of tripleo-docs and we could just link to tripleo.org docs from here.